### DISCUSSION TITLE
Quarkus 3.10 - Hibernate Search standalone POJO mapper, Flyway 10, security enhancements

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#40387] [Quarkus Blog/Website]
# Quarkus 3.10 - Hibernate Search standalone POJO mapper, Flyway 10, security enhancements

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-10-0-released/

**Comment by adampoplawski** [comment]
Also from 3.10.0 Quarkus Reactive messeging is supporting SQS :)

**Reply by prakashelango** [reply]
@adampoplawski, I see a preview banner on smallrye docs link. Is it good to use now? or any estimates on preview tenure
![image](https://github.com/quarkusio/quarkus/assets/28904745/53251998-b777-4019-9cbc-4a9754fc378b)

Is there a way to configure multiple sqs?

Thank you
